data/brands/amenity/restaurant.json

```diff
@@ -5004,6 +5004,17 @@
         "name": "Twin Peaks"
       }
     },
+    {
+      "displayName": "UDON",
+      "locationSet": {"include": ["001"]},
```

Comment:
UDON isn't a global franchise, they only operate in the following areas:

Spain
Portugal
República Dominicana
USA
Puerto Rico

Reply:
Or alternatively you could split this out in a separate PR as this one seems to be focussed on banks

Reply:
Corrected. I'm learning how to use github so sometimes I mistake and add all to the same PR 😅

Reply:
+1 for splitting it.